### PR TITLE
Updating API access URLs

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adds a section about API access URLs for cell-based accounts

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/cloud/about-cloud/regions-ip-addresses.md [hunk 1]
```diff
@@ -47,3 +47,17 @@ dbt Cloud, like many cloud services, relies on underlying AWS cloud infrastructu
...
+
+## API Access URLs
+
+dbt Cloud accounts with cell-based account prefixes have unique access URLs for account APIs. These URLs can be found in your **Account settings** below the **Account information** pane.
+
+<Lightbox src="/img/docs/dbt-cloud/access-urls.png" title="Access URLs in the account settings" />
+
+These URLs are unique to each account and begin with the same prefix as the URL used to [access your account](#accessing-your-account). The URLs cover the following APIs:
+
+- Semantic Layer JDBC API
```
Comment (runleonarun) [inline website/docs/docs/cloud/about-cloud/regions-ip-addresses.md:60]: I also see a URL to access the "Admin API" in the screenshot.

File: website/docs/docs/cloud/about-cloud/regions-ip-addresses.md [hunk 2]
```diff
@@ -47,3 +47,17 @@ dbt Cloud, like many cloud services, relies on underlying AWS cloud infrastructu
...
 
 * Optimizing VPN connections &mdash; You should integrate a proxy alongside VPN for users who leverage VPN connections. This strategy enables steady IP addresses for your connections, facilitating smooth traffic flow through the VPN and onward to dbt Cloud. By employing a proxy and a VPN, you can direct traffic through the VPN and then to dbt Cloud. It's crucial to set up the proxy if you need to integrate with additional services.
+
+## API Access URLs
+
+dbt Cloud accounts with cell-based account prefixes have unique access URLs for account APIs. These URLs can be found in your **Account settings** below the **Account information** pane.
+
+<Lightbox src="/img/docs/dbt-cloud/access-urls.png" title="Access URLs in the account settings" />
+
+These URLs are unique to each account and begin with the same prefix as the URL used to [access your account](#accessing-your-account). The URLs cover the following APIs:
```
Comment (runleonarun) [inline website/docs/docs/cloud/about-cloud/regions-ip-addresses.md:57]: In the screenshot, the Semantic Layer JDBC URL starts with `jdbc:arrow` not the same prefix as the others, Is this a typo or an outlier?
Screenshot 2024-05-17 at 3 17 11 PM

**Review by runleonarun — COMMENTED** [review]
@matthewshaver I have two questions, but otherwise looks good!

File: website/docs/docs/cloud/about-cloud/regions-ip-addresses.md (hunk 2 above)
Comment (matthewshaver) [reply website/docs/docs/cloud/about-cloud/regions-ip-addresses.md:57]: This is a unique format for the SL API

File: website/docs/docs/cloud/about-cloud/regions-ip-addresses.md (hunk 1 above)
Comment (matthewshaver) [reply website/docs/docs/cloud/about-cloud/regions-ip-addresses.md:60]: Adding it! Good catch

**Review by runleonarun — APPROVED** [review]
🚢 it!

**Pull request merged**